apollo-router/src/http_subgraph.rs

```diff
@@ -81,23 +81,26 @@ impl HttpSubgraphFetcher {
                 while let Some(next_chunk) = bytes_stream.next().await {
                     match next_chunk {
                         Ok(bytes) => {
-                            current_payload_bytes.extend(&bytes);
+                            tracing::debug_span!("aggregate_response_data").in_scope(|| {
```

Comment:
:grey_question: shouldn't this be placed on the whole while-loop?

Reply:
that would have made more sense. But now that https://github.com/apollographql/router/pull/286 is merged, it's one line :)

https://github.com/apollographql/router/pull/291/files#diff-866ee3b4571238b6af1680a2392a6b50068c10efe0cc8d94770a57fd572e2564R63